devito/symbolics/extended_sympy.py

```diff
@@ -170,6 +180,17 @@ def base(self):
         else:
             return self.pointer
 
+    @property
+    def bound_symbols(self):
+        # NOTE: this property requires the prefix '_' in front because
```

Comment:
But you are not using it right? the prefix `_`

Reply:
fixed